docs/docs-content/deployment-modes/agent-mode/install-agent-host.md

```diff
@@ -83,6 +83,13 @@ Palette. You will then create a cluster profile and use the registered host to d
   - [rsyslog](https://github.com/rsyslog/rsyslog). This is required for audit logs.
   - (Airgap only) [Palette Edge CLI](../../spectro-downloads.md#palette-edge-cli)
 
+  You can issue the following command to install all dependencies for installation (not including the Palette Edge CLI)
+  with the following command:
+
+  ```shell
+  sudo apt-get update && sudo apt-get install -y bash jq zstd rsync systemd-timesyncd conntrack iptables rsyslog --no-install-recommends
+  ```
+
```

Comment:
As Rocky Linux uses DNF as its package manager, do we need equivalent steps for that?

I'm also wary if Agent Mode scales to new OSes and we have to update this.

A suggestion could be to rephrase this to "For example, if using Ubuntu, you can issue the following com...".